pkgs/top-level/all-packages.nix

```diff
@@ -19577,7 +19577,7 @@ with pkgs;
 
   armadillo = callPackage ../development/libraries/armadillo { };
 
-  arrayfire = callPackage ../development/libraries/arrayfire { };
+  arrayfire = darwin.apple_sdk_11_0.callPackage ../development/libraries/arrayfire { };
```

Comment:
is this darwin.apple_sdk_11_0. actually correct? won't this fail on non-darwin?

Reply:
it works on non-darwin